ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/TextEmbeddingModel.java

```diff
@@ -40,13 +41,18 @@ public ModelTensorOutput predict(String modelId, MLInput mlInput) throws Transla
         for (String doc : textDocsInput.getDocs()) {
             Input input = new Input();
             input.add(doc);
+            if (mlParams instanceof AsymmetricTextEmbeddingParameters) {
+                AsymmetricTextEmbeddingParameters params = (AsymmetricTextEmbeddingParameters) mlParams;
+                input.add(AsymmetricTextEmbeddingParameters.SPARSE_EMBEDDING_FORMAT_FIELD, params.getSparseEmbeddingFormat().name());
```

Comment:
Based on this: https://github.com/opensearch-project/ml-commons/pull/3963#discussion_r2193685244, do we need to add null check for `params.getSparseEmbeddingFormat()`?